extensions/Khronos/KHR_lights/README.md

```diff
@@ -0,0 +1,143 @@
+
+### Defining Lights
+
+Lights are defined within an dictionary property in the glTF manifest file, by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_lights` property with a `lights` array inside it.
+
+Each light defines a mandatory `type` property that designates the type of light (`ambient`, `directional`, `point` or `spot`). The following example defines a white-colored directional light.
+
+```javascript
+"extensions": {
+    "KHR_lights" : {
+        "lights": [
+            {
+                "color": [
+                    1.0,
+                    1.0,
+                    1.0
+                ],
+                "type": "directional"
+            }
+        ]
+    }
+}
+```
+
+### Adding Light Instances to Nodes
+
+Nodes can have lights attached to them, just like any other objects that have a transform. Only lights of types `directional`, `point`, and `spot` have position and/or orientation so only these light types are allowed. These lights are attached to a node by defining the `extensions.KHR_lights` property and, within that, an index into the `lights` array using the `light` property.
+
+```javascript
+"nodes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+For light types that have a position (`point` and `spot` lights), the light's position is defined as the node's world location.
+For light types that have a direction (`directional` and `spot` lights), the light's direction is defined as the 3-vector `(0.0, 0.0, 1.0)` and the rotation of the node orients the light accordingly.
+
+### Adding Light Instances to Scenes
+
+Lights that have no transform information can be attached to scenes. `ambient` lights have no position, no orientation and no range. They are therefore global and belong to a scene rather than a node. These lights are attached to the scene by defining the `extensions.KHR_lights` property and, within that, an index into the `lights` array using the `light` property.
+
+```javascript
+"scenes" : [
+    {
+        "extensions" : {
+            "KHR_lights" : {
+                "light" : 0
+            }
+        }
+    }            
+]
+```
+
+### Light Types
+
+All light types share the common set of properties listed below.
+
+#### Light Shared Properties
+
+| Property | Description | Required |
+|:-----------------------|:------------------------------------------| :--------------------------|
+| `color` | RGB value for light's color in linear space. | No, Default: `[1.0, 1.0, 1.0]` |
+| `intensity` | Brightness of light in. The units that this is defined in depend on the type of light. `point` and `spot` lights use luminous intensity in candela (lm/sr) while `directional` lights use illuminance in lux (lm/m^2) | No, Default: `1.0` |
+| `type` | Declares the type of the light. | :white_check_mark: Yes |
```

Comment:
Perhaps you're missing `name` property here?
In `examples/lights.gltf` `light` has `name` property.

Reply:
Yes, yes I am. Thank you.

Reply:
Wait, actually, I think this is just a problem with my example. My intension was for lights to use the name of the node that they are attached to so the `name` in the examples shouldn't be there.

Reply:
About removing `name` entirely. I wish it weren't there on meshes or cameras, because in my mind the node _is_ a mesh or a camera, it doesn't _contain_ them, and so one of the two names must be thrown away in loading. But since both do have names in the current spec, it would be more consistent to keep one here. I'm undecided on this..

Reply:
Ah, you're right. I guess that's why I had it there. It is pretty odd to have a name on both the node and the light objects but, as it's consistent, I guess I'll put it back.